bibtex-actions.el

```diff
@@ -136,15 +133,22 @@ key associated with each one."
    (let* ((pdf (if (assoc "=has-pdf=" (cdr candidate)) " has:pdf"))
           (note (if (assoc "=has-note=" (cdr candidate)) "has:note"))
           (link (if (assoc "doi" (cdr candidate)) "has:link"))
-          (add (s-trim-right (s-join " " (list pdf note link)))))
+          (citekey (bibtex-completion-get-value "=key=" candidate))
+          (add (s-trim-right (s-join " " (list pdf note link))))
+          ; TODO: add separation between target and suffix
+          (suffix (bibtex-completion-format-entry candidate (1- (frame-width))
+                        bibtex-completion-display-formats-suffix)))
```

Comment:
This and the template setq are the only pieces of code dependent on the bibtex-completion PR. Could comment out and add a fixed placeholder for now.

Reply:
> This and the template setq are the only pieces of code dependent on the bibtex-completion PR. Could comment out and add a fixed placeholder for now.

Latest commit implements this. I'll wait a bit to see if Titus responds.

Reply:
The other alternative is to just add the two functions we need to do this ourselves, and then do this, so all of it is configurable.

``` elisp
(defcustom bibtex-actions-display-templates
  ; prefix will only work with rich ui, and on Emacs 28
  ; 'target' is the default entry display string
  '((prefix . '((t. "${=has-pdf=:1} ${=has-note=:1}  ")))
    (target . '((t . "${author:20}   ${title:48}   ${year:4}")))
    (suffix . '((t. "          ${=key=:15}    ${=type=:12}    ${tags:*}"))))
    "Configures display formatting for 'target', 'prefix' and 'suffix'.
The latter two are optional, but are used in the 'rich ui'."
    :group 'bibtex-actions
    :type  '(alist :key-type symbol :value-type function))
```